### PR TITLE
[RFR] Bootstrap project

### PR DESCRIPTION
**Pull request by Kmaschta**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+install:
+	npm install
+
+run:
+	nodemon server.js
+
+test:
+	@./node_modules/.bin/mocha
```
Comment (RobinBressan) [inline makefile]: Remove `@`, you have `make --silent` for that. Furthermore you can add the `node_modules/.bin` to the PATH in the makefile to make it more readable. Check others projects.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    },
+    "engine": "node >= 4.2.1",
+    "devDependencies": {
+        "eslint": "~1.5.1",
+        "eslint-config-airbnb": "^0.1.0",
+        "mocha": "^2.3.3"
+    },
+    "dependencies": {
+        "koa": "^1.1.1",
+        "koa-body-parser": "^1.1.2"
```
Comment (RobinBressan) [inline package.json]: replace all `^` by `~` to avoid major version update

File: server.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	var request = this.request;
+
+	console.log(request.method + ' ' + request.href);
+	console.log(request.headers);
+	console.log(request.body);
+
+	this.body = 'Hello world !';
+});
+
+app.listen(4000);
```
Comment (RobinBressan) [inline server.js]: Why `4000`? The standard is `3000`.

File: .gitignore [hunk 4]
```diff
@@ -25,3 +25,4 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+.eslintrc
```
Comment (fzaninotto) [inline .gitignore]: no, you should commit it

Comment (RobinBressan) [inline .gitignore]: indeed!

**Pull request merged**